package.json

```
@@ -9,27 +10,25 @@
     "glimmer",
     "prettier"
   ],
+  "repository": "git@github.com:dcyriller/ember-template-lint-plugin-prettier.git",
```

We should change this to

```suggestion
  "repository": "git@github.com:ember-template-lint/ember-template-lint-plugin-prettier.git",
```

Since release-it does read and use the repository info, sometimes this can cause a 307 from the API if the user doesn't have access. You'd be the only person with access to that repo.